build.js

```diff
@@ -20,13 +20,41 @@ fs.rm(join(__dirname, 'lib'), { force: true, recursive: true })
         {
           name: 'alias',
           setup({ onResolve }) {
+            // The yaml language service only imports re-exports of vscode-languageserver-types from
+            // vscode-languageserver.
+            onResolve({ filter: /^vscode-languageserver-textdocument$/ }, () => ({
+              path: 'vscode-languageserver-textdocument/lib/esm/main.js',
+              external: true,
+            }));
+            // The yaml language service only imports re-exports of vscode-languageserver-types from
+            // vscode-languageserver.
+            onResolve({ filter: /^vscode-languageserver$/ }, () => ({
+              path: 'vscode-languageserver-types/lib/esm/main.js',
+              external: true,
+            }));
+            // The yaml language service only imports re-exports of vscode-languageserver-types from
+            // vscode-languageserver.
+            onResolve({ filter: /^vscode-languageserver-types$/ }, () => ({
+              path: 'vscode-languageserver-types/lib/esm/main.js',
+              external: true,
+            }));
+            // The yaml language service uses path. We can stub it using path-browserify.
+            onResolve({ filter: /^path$/ }, () => ({
+              path: 'path-browserify',
+              external: true,
+            }));
```

Comment:
@fleon do you think it would be better to use this polyfill or to create a small filler? Only a small subset of `path` is needed.

Reply:
I think using `path` is fine for now. You could create another issue to create a small filler if that results in a smaller bundle size. But I would assume due to tree shaking, whatever functions we actually need only get included automatically.